README: restructure for contributor onboarding (#5)
kinvolk/contribution: README.md

# Contributions with Kinvolk

<img align="right" src="https://avatars2.githubusercontent.com/u/14073052?s=200&v=4" alt="Kinvolk" width="160">

> Here you will find information on contributing to Kinvolk projects, whether first time contribution, or referencing common guidlines (like the CoC).

Reviewer: ```suggestion
Here you will find information on contributing to Kinvolk projects, whether it's a first time contribution, or referencing common guidlines (like the CoC).
```

Reviewer 2: s/guidlines/guidelines/

Please read and uphold this code-of-conduct while participating in Kinvolk
projects and events.

## Projects

> There are many repos in the [Kinvolk](https://github.com/kinvolk/) org, though here is the highlights of projects.

Reviewer: ```suggestion
There are many repos in the [Kinvolk](https://github.com/kinvolk/) org. This section highlights the main projects.
```

### Flatcar Container Linux

- [Flatcar repositories](https://github.com/search?q=org%3Akinvolk+topic%3Aflatcar&type=repositories)
- [Flatcar docs](https://kinvolk.io/docs/flatcar-container-linux/latest)
- [Flatcar issue tracker](https://github.com/kinvolk/flatcar/issues)
- [contributing to Flatcar](https://github.com/kinvolk/Flatcar#contributing-to-flatcar-container-linux)
- chat: [#flatcar](irc://irc.freenode.org:6667/#flatcar) IRC channel on freenode.org

> - mailing-list: [flatcar-user](https://groups.google.com/g/flatcar-linux-user)

Reviewer: Nitpick: I'd not repeat the name of the project since the context is clear, but it's done now so :shrug: .

Author: I debated about that too :thinking:
